project/bugs.md

```diff
@@ -2,10 +2,10 @@
 
 The development team does a lot of testing. We have a whole test suite that is
 run frequently every day on numerous platforms in order to exercise all
-code and make sure everything works as supposed.
+code and make sure everything works as expected.
 
-Still, there are times when things are not working the way they should. Then
-we appreciate getting those problems reported.
+Still, there are times when things don't work the way they should, and we 
```

Comment:
We avoid contractions such as "don't" in this book: use "do not.

Reply:
ah, got it -- done!